uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api_test.js

```diff
@@ -304,6 +304,26 @@ define([
       });
     });
 
+    describe('delete()', function () {
+      var idToDelete = '123';
+
+      beforeEach(function () {
+        spyOn(LeaveRequestAPI, 'sendGET');
```

Comment:
When we test `API`, we should spy the GET call using `$httpBackend.whenGET` and return values from mocked data.